RoddyCore/src/de/dkfz/roddy/core/ExecutionContext.groovy

```diff
@@ -239,9 +238,9 @@ class ExecutionContext {
         if (workflowClass.name.endsWith('$py')) {
             // Jython creates a class called Workflow$py with a constructor with a single (unused) String parameter.
             workflow = (Workflow) workflowClass.getConstructor(String).newInstance("dummy")
+        } else {
```

Comment:
`if` previously didn't make sense, because result was directly overwritten. This is fixed now. Note that we experimented only once with Jython. Probably the feature won't be used anymore anyway.